packages/@uppy/transloadit/src/Assembly.ts

```diff
@@ -165,9 +179,11 @@ class TransloaditAssembly extends Emitter {
 
     try {
       this.#previousFetchStatusStillPending = true
-      const response = await this.#fetchWithNetworkError(
-        this.status.assembly_ssl_url,
-      )
+      const statusUrl = this.status.assembly_ssl_url ?? this.status.assembly_url
+      if (!statusUrl) {
+        throw new Error('Transloadit: Assembly status is missing a status URL')
+      }
```

Comment:
Is this ever undefined? I don't think so and in that case we're just add code for no reason.